Packs/Venafi/Integrations/VenafiV2/VenafiV2.yml

```diff
@@ -0,0 +1,312 @@
+category: IT Services
+commonfields:
+  id: Venafi TLS Protect
+  version: -1
+configuration:
+  - defaultvalue: "https://ao-tlspd.dev.ven-eco.com"
+    display: Server URL (e.g. https://192.168.0.1)
+    name: server
+    required: true
+    type: 0
+    section: Connect
+  - displaypassword: Password
+    name: credentials
+    required: true
+    type: 9
+    display: User Name
+    section: Connect
+  - display: Client ID
+    name: client_id
+    required: true
+    type: 0
+    section: Connect
+  - display: Trust any certificate (not secure)
+    name: insecure
+    type: 8
+    required: false
+  - display: Use system proxy settings
+    name: proxy
+    type: 8
+    required: false
+description: Retrieves information about certificates stored in Venafi.
+display: Venafi TLS Protect
+name: Venafi TLS Protect
+script:
+  commands:
+    - arguments:
+        - description: Specify YYYY-MM-DD or the ISO 8601 format.
+          name: CreatedOn
+          required: false
+        - description: Specify YYYY-MM-DD or the ISO 8601 format.
+          name: CreatedOnGreater
+          required: false
+        - description: Specify YYYY-MM-DD or the ISO 8601 format.
```

Comment:
But what is the definition of the argument?

CreatedOn - The date on which the certificated was created. Specify YYYY-MM-DD or the ISO 8601 format.
CreatedOnGreater - Find certificates created after this date. Specify YYYY-MM-DD or the ISO 8601 format.
CreatedOnLess - Find certificates created before this date. Specify YYYY-MM-DD or the ISO 8601 format.